templates/main.html

```diff
@@ -13,50 +13,48 @@
 		{{total}} articles published
 	</div>
 
-		<div class="n-storylines__timeline-container">
+	<div class="n-storylines__timeline-container">
 
 		{{#if children.0.children}}
-		<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
+			<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
 
 			{{#each children}}
-				<div class="n-storylines__heatmap-segment">
+			<div class="n-storylines__heatmap-segment">
 
-					<ul class="dot-container">
-						{{#if dot}}
-						{{#each dot}}
-						<li class='dot' style='margin-left: {{position}}%'>{{name}}</li>
-						{{/each}}
-						{{/if}}
-					</ul>
+				<ul class="dot-container">
+				{{#if dot}}
+					{{#each dot}}
+					<li class='dot' style='margin-left: {{position}}%'>{{name}}</li>
+					{{/each}}
+				{{/if}}
+				</ul>
 
-					<div class="n-storylines__heatmap-segment-colour" style="opacity: {{weight}};"></div>
-					<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
-				</div>
+				<div class="n-storylines__heatmap-segment-colour" style="opacity: {{weight}};"></div>
+				<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
+			</div>
 			{{/each}}
 		{{else if children}}
-
-		<div class="n-storylines__scroll">
+			<div class="n-storylines__scroll">
 			<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
 
-		{{#each children}}
+			{{#each children}}
 			<div class="n-storylines__heatmap-segment n-storylines__heatmap-segment--all-months ">
 				<ul class="dot-container">
 					{{#if dot}}
-					{{#each dot}}
-					<li class='dot' style="margin-left: {{position}}%">{{name}}</li>
-					{{/each}}
+						{{#each dot}}
+						<li class='dot' style="margin-left: {{position}}%">{{name}}</li>
+						{{/each}}
 					{{/if}}
 				</ul>
-						<div class="n-storylines__heatmap-segment-colour n-storylines__all-months" style="opacity: {{weight}};"></div>
-						<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
+				<div class="n-storylines__heatmap-segment-colour n-storylines__all-months" style="opacity: {{weight}};"></div>
+				<p class="n-storylines__heatmap-segment-name o-typography-aside__body--small">{{ name}}</p>
 			</div>
-		{{/each}}
+			{{/each}}
 		</div>
 		{{else}}
-		<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
+			<img class="n-storylines__back-btn" src="http://image.webservices.ft.com/v1/images/raw/fticon-v1:arrow-left?source=o-icons"/>
```

Comment:
Should be using the new image service url https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1:arrow-left?source=o-icons